configure.in: don't add the build host's /usr/include and /usr/lib to the 
search path of include files and libraries. This is not what you want when 
cross compiling.

Sent upstream as https://bugs.wireshark.org/bugzilla/show_bug.cgi?id=7926.

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
diff -Nuar wireshark-1.8.3.orig/configure.in wireshark-1.8.3/configure.in
--- wireshark-1.8.3.orig/configure.in	2012-08-15 23:33:32.000000000 +0300
+++ wireshark-1.8.3/configure.in	2012-10-28 10:07:54.864388531 +0200
@@ -635,29 +635,6 @@
 AC_SUBST(CORESERVICES_FRAMEWORKS)
 AC_SUBST(LAUNCHSERVICES_FRAMEWORKS)
 
-#
-# If using $prefix we add "$prefix/include" to the include search path
-# and "$prefix/lib" to the library search path.
-#
-if test "x$prefix" != "x" ; then
-	AC_MSG_CHECKING(whether to use $prefix for headers and libraries)
-	if test -d $prefix/include ; then
-		AC_MSG_RESULT(yes)
-		#
-		# Arrange that we search for header files in "$prefix/include", as
-		# various packages we use may have been installed under "$prefix/include".
-		#
-		CPPFLAGS="$CPPFLAGS -I$prefix/include"
-
-		#
-		# Arrange that we search for libraries in "$prefix/lib".
-		#
-		AC_WIRESHARK_ADD_DASH_L(LDFLAGS, $prefix/lib)
-	else
-		AC_MSG_RESULT(no)
-	fi
-fi
-
 dnl Look in /usr/local for header files and libraries ?
 dnl XXX FIXME don't include /usr/local if it is already in the system
 dnl search path as this causes gcc 3.2 on Linux to complain about a change
